msamp/nn/__init__.py

```diff
@@ -7,6 +7,8 @@
 from msamp.nn.module import ScalingModule
 from msamp.nn.state import model_state
 from msamp.nn.linear import FP8Linear, LinearReplacer
+from msamp.nn import functional
 from msamp.nn.clip_grad import clip_grad_norm_
+del functional
```

Comment:
Why "del functional"?

Reply:
I want to execute the overrider in functional.py but do not use it.

Reply:
Sure. It's acceptable.